ansible_collections/infoblox/b1ddi_modules/plugins/modules/b1_ipam_address_block.py

```diff
@@ -285,8 +285,6 @@ def main():
         host=dict(required=True, type='str'),
         address=dict(type='str'),
         space=dict(type='str'),
-        comment=dict(type='str'),
-        tags=dict(type='list', elements='dict', default=[{}]),
```

Comment:
May I know why we are removing the two field.

Reply:
Let me check - this may be in error/accidental

Reply:
I think these were removed by accident, these lines should remain.